F1WM/Repositories/NewsRepository.cs

```diff
@@ -79,6 +77,65 @@ public async Task<NewsDetails> GetNewsDetails(int id)
 			return news;
 		}
 
+		public async Task<NewsSummaryPaged> GetNewsByTagId(int? tagId, int page, int countPerPage)
+		{
+			await SetDbEncoding();
+			IQueryable<News> dbNews;
+
+			dbNews = context.NewsTagMatch
```

Comment:
No need to split dbNews declaration and assignment. You can just use `var dbNews = ...`